reana_client/cli/files.py

```diff
@@ -77,6 +76,7 @@ def get_files(ctx, workflow, _filter,
     Examples: \n
     \t $ reana-client ls --workflow myanalysis.42
     """
+    import tablib
```

Comment:
BTW another improvement like importing `tablib` is related to parsing `yaml`, which is very slow.  Do we need to use `FullLoader`? Cannot we use `CLoader`? It would be much faster:

```
In [19]: %timeit with open('./reana_client/schemas/reana_analysis_schema.json', 'r') as afile: res_f = yaml.load(afile, Loader=yaml.FullLoader)
24 ms ± 376 µs per loop (mean ± std. dev. of 7 runs, 10 loops each)

In [20]: %timeit with open('./reana_client/schemas/reana_analysis_schema.json', 'r') as afile: res_c = yaml.load(afile, Loader=yaml.CLoader)
1.87 ms ± 91.7 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)
```

Reply:
I tested it and didn't notice significant differences in performance. Perhaps `reana.yaml` should be bigger to notice them? Any example to better test it?